cuebot/src/main/resources/opencue.properties

```diff
@@ -44,6 +44,48 @@ dispatcher.job_frame_dispatch_max=8
 # Maximum number of frames to dispatch from a host at one time.
 dispatcher.host_frame_dispatch_max=12
 
+# Number of threads to keep in the pool for launching job.
+dispatcher.launch_queue.core_pool_size=1
+# Maximum number of threads to allow in the pool for launching job.
+dispatcher.launch_queue.max_pool_size=1
+# Queue capacity for launching job.
+dispatcher.launch_queue.queue_capacity=100
+
+# Number of threads to keep in the pool for various operation.
+dispatcher.dispatch_pool.core_pool_size=4
+# Maximum number of threads to allow in the pool for various operation.
+dispatcher.dispatch_pool.max_pool_size=4
+# Queue capacity for various operation.
+dispatcher.dispatch_pool.queue_capacity=500
+
+# Number of threads to keep in the pool for management operation.
+dispatcher.manage_pool.core_pool_size=8
+# Maximum number of threads to allow in the pool for management operation.
+dispatcher.manage_pool.max_pool_size=8
+# Queue capacity for management operation.
+dispatcher.manage_pool.queue_capacity=250
+
+# Number of threads to keep in the pool for handling Host Report.
+dispatcher.report_queue.core_pool_size=6
+# Maximum number of threads to allow in the pool for handling Host Report.
+dispatcher.report_queue.max_pool_size=8
+# Queue capacity for handling Host Report.
+dispatcher.report_queue.queue_capacity=1000
+
+# Number of threads to keep in the pool for kill frame operation.
+dispatcher.kill_queue.core_pool_size=6
+# Maximum number of threads to allow in the pool for kill frame operation.
+dispatcher.kill_queue.max_pool_size=8
+# Queue capacity for kill frame operation.
+dispatcher.kill_queue.queue_capacity=1000
```

Comment:
The original values
https://github.com/AcademySoftwareFoundation/OpenCue/blob/c93e8629f641a95f47617c50865f7e316e94f5c8/cuebot/src/main/java/com/imageworks/spcue/dispatcher/HostReportQueue.java#L35-L37